src/main/java/com/google/api/codegen/transformer/ruby/RubySurfaceNamer.java

```diff
@@ -203,22 +209,23 @@ public String getRequestTypeName(ModelTypeTable typeTable, TypeRef type) {
   }
 
   @Override
-  public List<String> getThrowsDocLines(GapicMethodConfig methodConfig) {
+  public List<String> getThrowsDocLines(MethodConfig methodConfig) {
     return ImmutableList.of("@raise [Google::Gax::GaxError] if the RPC is aborted.");
   }
 
   @Override
   public List<String> getReturnDocLines(
-      GapicInterfaceContext context, GapicMethodConfig methodConfig, Synchronicity synchronicity) {
-    Method method = methodConfig.getMethod();
+      InterfaceContext context, MethodConfig methodConfig, Synchronicity synchronicity) {
+    Method method = ((GapicMethodConfig) methodConfig).getMethod();
```

Comment:
Put in a TODO for this, because the next PR will abstract away the API source type from method models.